integration/presto/src/main/java/org/apache/carbondata/presto/readers/BooleanStreamReader.java

```diff
@@ -0,0 +1,31 @@
+package org.apache.carbondata.presto.readers;
+
+import java.io.IOException;
+
+import com.facebook.presto.spi.block.Block;
+import com.facebook.presto.spi.block.BlockBuilder;
+import com.facebook.presto.spi.block.BlockBuilderStatus;
+import com.facebook.presto.spi.type.Type;
+
+public class BooleanStreamReader implements StreamReader {
```

Comment:
Can you explain why add the class BooleanStreamReader , it is for which purpose?